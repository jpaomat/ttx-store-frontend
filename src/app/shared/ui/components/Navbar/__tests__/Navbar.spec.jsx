const Renderer = require('react-test-renderer/shallow');
const { Navbar } = require('../Navbar');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<Navbar />);
		expect(component).toMatchSnapshot();
	});
});