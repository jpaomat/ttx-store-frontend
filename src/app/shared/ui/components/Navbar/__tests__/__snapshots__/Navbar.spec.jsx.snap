// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<nav
  className="navbar"
>
  <Link
    className="navbar-brand"
    to="/"
  >
    <div
      className="navbar-brand__logo"
    >
      <LogoTtx />
    </div>
  </Link>
  <div
    className="navbar-collapse"
  >
    <ul
      className="navbar-nav-actions"
    >
      <li
        className="navbar-nav-actions__cart"
      >
        <div
          className="navbar-nav-actions__cart-content"
        >
          <a
            className="navbar-nav-actions__shopingCart"
          >
            <ShoppingCart
              items={20}
            />
          </a>
        </div>
      </li>
    </ul>
  </div>
</nav>
`;
